package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/ImportHelpers.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using USD.NET;
+using USD.NET.Unity;
+using Object = UnityEngine.Object;
+
+namespace Unity.Formats.USD
+{
+    public static class ImportHelpers
+    {
+        public static GameObject ImportSceneAsGameObject(Scene scene, SceneImportOptions importOptions = null)
+        {
+            string path = scene.FilePath;
+
+            // Time-varying data is not supported and often scenes are written without "Default" time
+            // values, which makes setting an arbitrary time safer (because if only default was authored
+            // the time will be ignored and values will resolve to default time automatically).
+            scene.Time = 1.0;
+
+            if (importOptions == null)
+            {
+                importOptions = new SceneImportOptions();
+                importOptions.usdRootPath = GetDefaultRoot(scene);
+            }
+
+            GameObject root = new GameObject(GetObjectName(importOptions.usdRootPath, path));
+
+            if (Selection.gameObjects.Length > 0)
+            {
+                root.transform.SetParent(Selection.gameObjects[0].transform);
+            }
+
+            try
+            {
+                UsdToGameObject(root, scene, importOptions);
+                return root;
+            }
+            catch (SceneImporter.ImportException)
+            {
+                Object.DestroyImmediate(root);
```

Comment:
Not sure if it makes a difference: in playmode you wanna call Destroy, and out of playmode DestroyImmediate. Make yourself an utility function that abstracts this.